axon-server/administration/reset-event-processor-token.md

```diff
@@ -0,0 +1,146 @@
+Now, when `resetTokens()` is called, the executing instance can temporarily claim all segments, reset the tokens and store the new values.
+After the token has been reset, the event processor can be started again.
+
+Note that older versions of axon framework (pre 4.6) directly acknowledge a received admin instruction, without waiting for its execution.
+This might require manually waiting for an EventProcessor to be fully stopped before the token can be reset.
+A possible approach for this is shown in the last section of this page.
+
+## Using Axon Framework
+Axon Framework exposes all required functionality for a reset in the `StreamingEventProcessor` class. In the following, we provide a sample for its usage. For more details, read the documentation on the framework classes [here](../../axon-framework/events/event-processors/streaming.md#triggering-a-reset).
+Instances of `StreamingEventProcessor` can be obtained by querying the `eventProcessingConfiguration` method of your global configuration.
+```java
+configuration.eventProcessingConfiguration()
+    .eventProcessorByProcessingGroup(processorName,
+        StreamingEventProcessor.class)
+    ...
+```
+Calling `shutDown`, `resetTokens` and `start` on the retrieved `StreamingEventProcessor` performs the required steps to reset its token.
+This can be achieved as shown in the following code snippet:
+```java
+configuration.eventProcessingConfiguration()
+    .eventProcessorByProcessingGroup(processorName,
+        StreamingEventProcessor.class)
+    .ifPresent(streamingEventProcessor -> {
+        if (streamingEventProcessor.supportsReset()) {
+            streamingEventProcessor.shutDown();
+            streamingEventProcessor.resetTokens();
+            streamingEventProcessor.start();
+        }
+    });
+```
+
+Note that this only concerns stopping the local instances. 
+If there are instances running on other nodes, you either need to use AxonServer or build a solution for this yourself.
+For more details, refer to the section in the Axon Framework [reference guide](../../axon-framework/events/event-processors/streaming.md#triggering-a-reset).
+
+## Using the REST API
+Axon Server exposes a REST API to pause and start EventProcessors.
+When used in conjunction with the `resetTokens` method from earlier, this can be used to make sure that all instances of an EventProcessor are paused before a token is reset.
+The required parameters to do this are the following:
+- *component*: name of the component the EventProcessor is part of
+- *processor*: name of the EventProcessor itself
+- *context*: name of the context for which to reset the tokens
+- *tokenStoreId*: the identifier used to distinguish the desired token from other tokens stored in the same store. 
+
+Now 
+`/v1/components/{component}/processors/{processor}/pause?context={context}&tokenStoreIdentifier={tokenStoreId}`
+can be called, causing AxonServer to request all matched EventProcessors to stop.
```

Comment:
Similar "space or no space" argument as with Event Processors. Furthermore, you're using "Axon Server" at the start of this section. 

Personally, I opt for using a space between Axon and Server.
But, consistency is more important than my personal preference here if you ask me.